### ISSUE TITLE
Scholarly sending wrong url to Scraper API

### ISSUE THREAD

**Issue by eduardozf** [scholarly-python-package/scholarly#307]
I noticed that when used citedby, scholarly send the wrong url to Scraper API

My code:
```
from scholarly import scholarly, ProxyGenerator

pg = ProxyGenerator()
pg.ScraperAPI('MY_EXEMPLE_KEY')
scholarly.use_proxy(pg)

search_query = scholarly.search_author('Steven A Cholewiak')
author = scholarly.fill(next(search_query))
print(author)

# Print the titles of the author's publications
print([pub['bib']['title'] for pub in author['publications']])

# Take a closer look at the first publication
pub = scholarly.fill(author['publications'][0])
print(pub)

# Which papers cited that publication?
print([citation['bib']['title'] for citation in scholarly.citedby(pub)])
```

The requests through my proxy: (Requests works until the scholarly.fill, then starts to fail)
![Screenshot_2](https://user-images.githubusercontent.com/42107537/126387602-023cc49c-05c6-41b9-a451-3d0ad20316e5.png)

**Comment by ipeirotis** [comment]
Indeed, the problem is at https://github.com/scholarly-python-package/scholarly/blob/master/scholarly/_navigator.py#L105

By changing the pagerequest

> pagerequest = **'http://api.scraperapi.com'

when we have a failure, the original pagerequest has been disappeared.

**Comment by ipeirotis** [comment]
I guess a solution is to use the proxy approach from Scraper API, instead of their API (https://www.scraperapi.com/documentation/)

**Comment by ipeirotis** [comment]
Should be fixed in v1.2.2.
